aten/src/ATen/native/quantized/cuda/base_functions.cu

```diff
@@ -0,0 +1,34 @@
+#include <ATen/Dispatch.h>
+#include <ATen/native/DispatchStub.h>
+#include <ATen/native/ReduceOps.h>
+#include <ATen/native/SharedReduceOps.h>
+#include <ATen/native/TensorIterator.h>
+#include <ATen/native/cuda/Reduce.cuh>
+
+namespace at {
+namespace native {
+
+bool cuda_equal_quantized(const Tensor& self, const Tensor& src) {
```

Comment:
Are we checking for equality of quantized tensors here? I think we should also check the qscheme and scale/zero_point in that case.
cc @jerryzh168 looked at something similar for all_close for quantized tensors